dev/VSIX/ProjectTemplates/Desktop/CSharp/PackagedApp/WapProj/WapProjTemplate.wapproj

```diff
@@ -77,28 +77,15 @@
     </ProjectReference>
   </ItemGroup>
 
-  <PropertyGroup>
-    <!--PackageReference.GeneratePathProperty does not support NUGET_PACKAGES env var...-->
-    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)'==''">$(NUGET_PACKAGES)</NuGetPackageRoot>
-    <NuGetPackageRoot Condition="'$(NuGetPackageRoot)'==''">$(UserProfile)\.nuget\packages</NuGetPackageRoot>
-    <PkgMicrosoft_ProjectReunion Condition="'$(PkgMicrosoft_ProjectReunion)'==''">$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'Microsoft.ProjectReunion', '$ext_ReunionNupkgVersion$'))</PkgMicrosoft_ProjectReunion>
-    <PkgMicrosoft_ProjectReunion Condition="!Exists($(PkgMicrosoft_ProjectReunion))">$(SolutionDir)packages\Microsoft.ProjectReunion.$ext_ReunionNupkgVersion$\</PkgMicrosoft_ProjectReunion>
-    <PkgMicrosoft_ProjectReunion_WinUI Condition="'$(PkgMicrosoft_ProjectReunion_WinUI)'==''">$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', 'Microsoft.ProjectReunion.WinUI', '$ext_ReunionWinUINupkgVersion$'))</PkgMicrosoft_ProjectReunion_WinUI>
-    <PkgMicrosoft_ProjectReunion_WinUI Condition="!Exists($(PkgMicrosoft_ProjectReunion_WinUI))">$(SolutionDir)packages\Microsoft.ProjectReunion.WinUI.$ext_ReunionWinUINupkgVersion$\</PkgMicrosoft_ProjectReunion_WinUI>
-    <Microsoft_ProjectReunion_AppXReference_props>$([MSBuild]::NormalizeDirectory('$(PkgMicrosoft_ProjectReunion)', 'build'))Microsoft.ProjectReunion.AppXReference.props</Microsoft_ProjectReunion_AppXReference_props>
-    <Microsoft_WinUI_AppX_targets>$([MSBuild]::NormalizeDirectory('$(PkgMicrosoft_ProjectReunion_WinUI)', 'build'))Microsoft.WinUI.AppX.targets</Microsoft_WinUI_AppX_targets>
-  </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.ProjectReunion" Version="[$ext_ReunionNupkgVersion$]" GeneratePathProperty="true">
-      <ExcludeAssets>all</ExcludeAssets>
+    <PackageReference Include="Microsoft.ProjectReunion" Version="[$ext_ReunionNupkgVersion$]">
+      <IncludeAssets>build</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.ProjectReunion.WinUI" Version="[$ext_ReunionWinUINupkgVersion$]" GeneratePathProperty="true">
-      <ExcludeAssets>all</ExcludeAssets>
+    <PackageReference Include="Microsoft.ProjectReunion.WinUI" Version="[$ext_ReunionWinUINupkgVersion$]">
```

Comment:
do we need WinUI reference?

Reply:
no, not needed, but benign.  if you create a wapproj from a template, and add a reunion package reference, you only see the metapackage.  but all thin packages are also added.